src/GoTrueClient.ts

```diff
@@ -330,6 +335,39 @@ export default class GoTrueClient {
     return this.currentSession
   }
 
+  async getSession(): Promise<
```

Comment:
Note - please document the function calls (the docs are auto-generated based on the TSDoc)